playground/terraform/infrastructure/cloudbuild-manual-setup/README.md

```diff
@@ -0,0 +1,156 @@
+    - Service Account Admin
+    - Storage Admin
+    - Service Usage Admin
+    - Cloud Build Editor
+    - Security Admin
+    - Service Account User
+- [gcloud CLI](https://cloud.google.com/sdk/docs/install-sdk)
+- An existing GCP Bucket to save Terraform state - `state-bucket`
+- DNS name for your Playground deployment instance
+- [Terraform](https://www.terraform.io/)
+- [Apache Beam GitHub](https://github.com/apache/beam) repository cloned locally
+
+## 1. Set up the Google Cloud Build for your GCP project
+
+The `playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup` provisions dependencies required to set up Cloud Build for Playground:
+- Required API services
+- Cloud Build service account - `playground-cloudbuild-sa`
+- IAM permissions for Cloud Build service account - `playground-cloudbuild-sa`
+
+#### To execute the module:
+
+1. Set following environment variables:
+   - `STATE_BUCKET`: GCP Storage bucket name to save Terraform state
+   - `GOOGLE_PROJECT`: GCP Project ID
+   - `GOOGLE_REGION`: GCP region to save triggers to
+   - `PLAYGROUND_REGION`: GCP Region (us-central1) where playground infrastructure will be deployed
+   - `PLAYGROUND_LOCATION`: GCP Location (us-central1-b) where playground infrastructure will be deployed
+   - `ENVIRONMENT_NAME`: Environment where Playground will be deployed (located at playground/terraform/environment/environment_name)
+   - `DNS_NAME`: DNS for deployed Playground webpage
+   - `NETWORK_NAME`: GCP VPC Network Name for Playground deployment
+   - `GKE_NAME`: Playground GKE Cluster name
+   - `TAG`: Tag for Playground images
+   - `SDK_TAG`: Tag for SDK images of Apache Beam Go and Python (typically last stable 2.42.0, 2.43.0)
+   - `DOCKER_REPOSITORY_ROOT`: GCP Artifact Registry repository name to store Playground container images
+
+```console
+    export STATE_BUCKET="state-bucket" \
+    GOOGLE_PROJECT="project-id" \
+    GOOGLE_REGION="us-central1" \
+    PLAYGROUND_REGION="us-central1" \
+    PLAYGROUND_LOCATION="us-central1-a" \
+    ENVIRONMENT_NAME="env-name" \
+    DNS_NAME="dns-name" \
+    NETWORK_NAME="network-name" \
+    GKE_NAME="gke-cluster-name" \
+    TAG="tag-name" \
+    SDK_TAG="2.43.0"
+
+    export DOCKER_REPOSITORY_ROOT=\
+    "$PLAYGROUND_REGION-docker.pkg.dev/$GOOGLE_PROJECT/playground-repository"
+```
+**Note:**  Please see [Cloud Build locations](https://cloud.google.com/build/docs/locations) for the list of all supported locations.
+
+2. Run commands:
+
+
+```console
+# Create a new authentication configuration for GCP Project with the created user account
+gcloud init
+
+# Command imports new user account credentials into Application Default Credentials
+gcloud auth application-default login
+
+# Navigate to 01.setup folder
+cd playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup/
+
+# Run terraform scripts
+terraform init -backend-config="bucket=$STATE_BUCKET"
+terraform apply -var "project_id=$GOOGLE_PROJECT"
```

Comment:
`terraform apply -var="project_id=$(gcloud config get-value project)"`

Reply:
Changed to proposed